app/uk/gov/hmrc/gform/core/parsers/ValueParser.scala

```diff
@@ -397,6 +397,8 @@ trait ValueParser extends RegexParsers with PackratParsers with BasicParsers {
     str
   }
 
+  lazy val notEmpty: Parser[String] = """.+""".r ^^ { str => str }
```

Comment:
Do we really need to add this regex? I think we should accept only Alphabetic chars.

Reply:
We should think how to handle empty values. It returns a parser error. 
```           {
              "en": "Blind Persons Allowance",
              "value": ""
            },```

Reply:
Ok thanks, I'll look into this!

Reply:
Hi @DursunGundogan, I've updated it to allow any String based values to the template validator and added an additional check for empty string or spaces.